pkg/tools/etcd_helper.go

```diff
@@ -475,35 +474,22 @@ func (h *EtcdHelper) PrefixEtcdKey(key string) string {
 	return path.Join("/", h.PathPrefix, key)
 }
 
-// GetEtcdVersion performs a version check against the provided Etcd server, returning a triplet
-// of the release version, internal version, and error (if any).
-func GetEtcdVersion(host string) (releaseVersion, internalVersion string, err error) {
+// GetEtcdVersion performs a version check against the provided Etcd server,
+// returning the string response, and error (if any).
+func GetEtcdVersion(host string) (string, error) {
 	response, err := http.Get(host + "/version")
 	if err != nil {
-		return "", "", err
+		return "", err
 	}
 	defer response.Body.Close()
-
-	body, err := ioutil.ReadAll(response.Body)
-	if err != nil {
-		return "", "", err
-	}
-
-	var dat map[string]interface{}
-	if err := json.Unmarshal(body, &dat); err != nil {
-		return "", "", fmt.Errorf("unknown server: %s", string(body))
+	if response.StatusCode >= 400 {
```

Comment:
Okay http.StatusOK sounds good for now. If we need to handle others we can think about it later.